simple_server/handle_client.cpp

```diff
@@ -9,9 +9,13 @@
 #include <iostream>
 #include <unistd.h>
 
+#include "../inc/Message.hpp"
+
+
 void	handle_client(int sd, int &max_sd, fd_set &master_set)
 {
-	char	buffer[80];
+	Message	msgP;
```

Comment:
If class is named Message, then variable should be named message.
If class is named Msg, then variable should be named msg.
This way, it's clearer that there is not another class with a similar name.